core/shared/src/main/scala/scalaz/zio/ExitResult.scala

```diff
@@ -94,12 +83,22 @@ sealed abstract class ExitResult[+E, +A] extends Product with Serializable { sel
   /**
    * Folds over the value or cause.
    */
-  final def fold[Z](completed: A => Z, failed: Cause[E] => Z): Z =
+  final def fold[Z](failed: Cause[E] => Z, completed: A => Z): Z =
```

Comment:
👍 Makes more sense! Also I think the `toEither` in `unsafeRun` could be replaced with:
```scala
def unsafeRun[E, A](io: IO[E, A]): A = unsafeRunSync(io).leftMap(FiberFailure).fold(throw _, identity)
```